app/views/sections/_form.html.erb

```diff
@@ -1,79 +1,116 @@
-<div class='row'>
-  <div class="col-md-8">
+<% content_for :head do %>
+  <%= javascript_include_tag "vendor/jquery-1.11.0.min" %>
+  <%= javascript_include_tag "ajax_setup" %>
+  <%= javascript_include_tag "markdown_preview" %>
```

Comment:
What do these JavaScript files do? Are they needed in the Design System layout, or should they be in `_legacy_form.html.erb`?

While porting pages across to the Design System, we've generally tried to avoid including jQuery and legacy JavaScript. Instead, we've used govuk_publishing_components that implement the desired behaviour, or have used vanilla JS for implementation.

Reply:
at the moment we're still relying on the old govspeak preview, which requires these js tags. to upgrade to the new govspeak needs implementing a markdown preview endpoint, unless theres another way?

Reply:
The jquery stuff has been removed from main, and this PR's changes rebased on top of that, so it no longer has any dependency on jquery on design system pages.